packages/studio/src/components/ErrorComponentPreview.tsx

```diff
@@ -1,14 +1,14 @@
-import { ErrorComponentState } from "@yext/studio-plugin";
 import { useCallback, useMemo, useRef, useState } from "react";
 import { v4 } from "uuid";
 import ErrorBoundary from "./common/ErrorBoundary";
-import { Tooltip } from "react-tooltip";
+import { ErrorComponentState } from "@yext/studio-plugin";
 
 export default function ErrorComponentPreview(props: {
   errorComponentState: ErrorComponentState;
   element: JSX.Element | null;
+  setTooltipProps;
```

Comment:
nit: can we add a type here (and other places we have setTooltipProps as a prop)?

Reply:
fixed + done!